### ISSUE TITLE
Can't record framer projects

### ISSUE THREAD

**Issue by jasonLaster** [replayio/gecko-dev#221]
For some reason framer projects do not open when recording. There might be a perf issue where it takes forever, but i waited ~5 mins

STR
1. go to https://framer.com/projects/Page-Example--f7neSA5Vq2Wzxue8KswY-dGSUT?node=RHMYCj8YE-page
2. login as test@replay.io / ReplayTest123
3. record

**Comment by jasonLaster** [comment]
hmm, just saw that carlmetta recorded some framer pages:

https://replay.io/view?id=f254db8f-5f4b-40c0-af7b-899c6c7c3610
https://replay.io/view?id=ea212154-b412-46f7-8867-e9400f5d75ff
https://replay.io/view?id=50196f4d-1364-41dc-88ff-7ae36729ff3f

interestingly, i still cannot

**Comment by bhackett1024** [comment]
I'm able to reproduce this.  I opened the console in the recording tab and there is a "ReferenceError: WebAssembly is not defined" error.  If I go to about:config and set javascript.options.wasm to false, the page doesn't load in a non-recording tab either.

So, the problem seems to be that we disable wasm when recording/replaying.  I think Carlmetta was able to make those recordings by recording different parts of the framer site that don't completely break when wasm is disabled.

We actually have a secret switch to allow recording/replaying pages with wasm content (I added this switch earlier this year so we could record/replay earlier versions of the devtools that had some wasm code to deal with source maps).  If I launch replay from the terminal with the RECORD_REPLAY_ALLOW_WASM=1 env var set, we can record this page.  I just created the recording below that way.

https://replay.io/view?id=a8a745dc-5270-4aad-bea5-332556dfc5ba

We don't enable this switch by default because the wasm signal handlers will not be turned on, which is a security hazard.  That wouldn't be hard to fix --- we could install the signal handlers when recording but not when replaying, and if they ever trip while recording we just invalidate the recording.
